app/views/activitypub/types/announce.activitystreams2.rabl

```diff
@@ -0,0 +1,3 @@
+extends 'activitypub/intransient.activitystreams2.rabl'
+
+node(:type) { 'Announce' }
```

Comment:
ActivityPub has no verb for "share"?

Reblogs should definitely be "share [original note]", I don't feel like "announce [creation of original note]" is semantic at all